notebooks/246-paint-by-example/246-paint-by-example.ipynb

```diff
@@ -0,0 +1,943 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Paint By Example: Exemplar-based Image Editing with Diffusion Models\n",
+    "\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Stable Diffusion in Diffusers library\n",
+    "To work with Stable Diffusion, we will use the Hugging Face [Diffusers](https://github.com/huggingface/diffusers) library. To experiment with in-painting we can use Diffusers which exposes the [StableDiffusionInpaintPipeline](https://huggingface.co/docs/diffusers/using-diffusers/conditional_image_generation) similar to the [other Diffusers pipelines](https://huggingface.co/docs/diffusers/api/pipelines/overview). The code below demonstrates how to create `StableDiffusionInpaintPipeline` using `stable-diffusion-2-inpainting`.\n",
+    "To create the drawing tool we will install Gradio for handling user interaction.\n",
+    "\n",
+    "This is the overall flow of the application:\n",
+    "![Flow Diagram](https://user-images.githubusercontent.com/103226580/236954918-f364b227-293c-4f78-a9bf-9dcebcb1034a.png)\n",
+    "\n",
+    "This is the detailed flowchart for the pipeline:\n",
+    "![pipeline-flowchart](https://github.com/openvinotoolkit/openvino_notebooks/assets/103226580/cde2d5c4-2540-4a45-ad9c-339f7a69459d)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install gradio\n",
+    "%pip install -q \"diffusers>=-1.14.0\" \"openvino>=2023.1.0\" openvino-dev \"transformers >= 4.25.1\""
```

Comment:
`%pip install -q \"diffusers>=-1.14.0\" \"openvino>=2023.2.0\" openvino-dev \"transformers >= 4.25.1\"`

Reply:
Done. 
OpenVino is now set to 2023.2.0